src/main/java/mail/lemeshev/mail_server/MailServerApplication.java

```diff
@@ -0,0 +1,12 @@
+package mail.lemeshev.mail_server;
```

Comment:
Underscores should not be used in package names.
See:
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names